starter-code/src/index.js

```diff
@@ -3,6 +3,18 @@ import ReactDOM from "react-dom";
 import App from "./App.js";
 
 ReactDOM.render(
-  <App />,
+  <section>
```

Comment:
There are exceptions but normally we don't put additional code in here. All our code goes in ``` <App>   ```  and it's children (eg components). This will become clearer in the lessons this week